whitepaper_templates/whitepaper-template.md

```diff
@@ -0,0 +1,45 @@
+# Title
+
+## OASIS Open Project : [Coalition for Secure AI (CoSAI)](https://github.com/cosai-oasis) \[Workstream name\](hyperlink to remember to update Title and Author in document Properties \!\!\!)
+
+## Additional artifacts: This document is one component of a Work Product that also includes: XML schemas: (list file names or directory name) Other parts (list titles and/or file names or directory name)
+
+## Abstract:
+
+## Summary of the technical purpose of the document.
+
+## Status:
+
+## 1\. Introduction
+
+## 2\. Section Title
+
+### 2.1 Level 2 Section Title
+
+#### **2.1.1 Level 3 Section Title**
+
+##### **2.1.1.1 Level 4 Section Title**
+
+###### *2.1.1.1.1 Level 5 Section Title*
+
+Note: Avoid using more than five heading levels.
+
+## 3\. Takeaways and Conclusion
+
+## 4\. References
+
+## 6\. Acknowledgements
+
+## Workstream Leads Chairs: WS Lead Chair Name ([Chair.Name@example.com](mailto:Chair.Name@example.com)), Example Corp. (mailto: link for email address; http:// link for affiliation web site) (remove "s" from Chairs if one)
```

Comment:
Can we please remove the terms "Chairs" and "Chair"? The term used is Workstream Lead throughout the governance.